packages/web-components/src/components/page-header/page-header.stories.ts

```diff
@@ -58,7 +63,29 @@ const tags = [
 export const Default = {
   render: () => html`
     <cds-page-header>
-      <cds-page-header-breadcrumb>BreadcrumbBar</cds-page-header-breadcrumb>
+      <cds-page-header-breadcrumb border>
```

Comment:
```suggestion
      <cds-page-header-breadcrumb>
```
since `border` default value is true, no need to apply here